ext/mysqli/mysqli.stub.php

```diff
@@ -76,7 +77,7 @@ public function dump_debug_info() {}
      * @return bool
      * @alias mysqli_debug
      */
-    public function debug(string $options) {}
+    public static function debug(string $options) {}
```

Comment:
On second consideration, I would not make this one static. It doesn't buy you anything in terms of usage (it's unlikely you'd use `mysqli::debug()` without also having a mysqli instance) and there is a conceivable universe where debug options could be set per connection in the future...

Reply:
Ok, makes sense!